public/v4/apps/gotify.yml

```diff
@@ -0,0 +1,26 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: gotify/server:$$cap_version
+        volumes:
+            - '$$cap_appname-data:/app/data'
+            - '$$cap_appname-config:/etc/gotify/'
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
```

Comment:
As really tiny advice I will more prefer a variable called cap_gotify_version but anyway seems to work, so... cool!!

Reply:
thanks for the advice, I will do it next time :)